app/javascript/chat/ChatChannelSettings/MembershipManager/MembershipSection.jsx

```diff
@@ -32,17 +34,7 @@ export const MembershipSection = ({
 };
 
 MembershipSection.propType = {
-  memberships: PropTypes.arrayOf(
-    PropTypes.shape({
-      name: PropTypes.string.isRequired,
-      membership_id: PropTypes.number.isRequired,
-      user_id: PropTypes.number.isRequired,
-      role: PropTypes.string.isRequired,
-      image: PropTypes.string.isRequired,
-      username: PropTypes.string.isRequired,
-      status: PropTypes.string.isRequired,
-    }),
-  ).isRequired,
+  memberships: PropTypes.arrayOf(defaultMembershipPropType).isRequired,
```

Comment:
Nitpick (blocking): Move this change to a separate pull request as it's not related to adding missing keys in mapped components.